app/middleware/exception_handlers.py

```diff
@@ -1,28 +1,53 @@
-import fastapi
+import traceback
 
+import fastapi
 from app.utils import context_vars
 from logging import getLogger
+from app.utils.errors import error_messages
 
 logger = getLogger(__name__)
 request_id_context = context_vars.request_id_context
 
 
-async def http_exception_handler(request, exc):
+async def validation_exception_handler(
+    request: fastapi.Request, exc: fastapi.exceptions.RequestValidationError
+):
+    error_details = exc.errors()
+
+    first_error = error_details[0]
+    loc = first_error.get("loc", [])
+    expected = first_error.get("type", "unknown")
```

Comment:
In the validation errors the attribute `type` doesn't seem to fit the "expected" descriptor, it usually refers to the type of error, for example: in `GET /metrics/LossPersistence/values?area_type=ea&area` the validation error is that "area_id is required", but type is `missing`, another example: in `POST /metrics/LossPersistence/values` if the bbox is missing, type is `value_error` which doesn't say anything about the expected input.

I suggest to remove the whole "expected" part